### PR TITLE
Remove SCP's XDR files

### PR DESCRIPTION
**Pull request by AndrejMitrovic**
We can just look at the C++ headers, they're not hard to follow and they show any additional generated code we may have to look for.

### CODE REVIEW

**Comment by Geod24** [comment]
I suppose this is because you want to remove some stuff from the header files ? I originally kept them as they make comparing with upstream easier.

**Comment by AndrejMitrovic** [comment]
Ah if it helps with upstream diff, then we could keep them. I thought they were here by mistake.
